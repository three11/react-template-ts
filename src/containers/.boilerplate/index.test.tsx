import * as React from 'react';
import { shallow } from 'enzyme';

import { Page } from '.';

jest.mock('react-redux', () => ({
	connect: (): jest.Mock => jest.fn(),
	useSelector: jest.fn(),
	useDispatch: (): jest.Mock => jest.fn()
}));

describe('Page component', () => {
	it('should render successfully', () => {
		const tree = shallow(<Page />);

		expect(tree).toMatchSnapshot();
	});
});
