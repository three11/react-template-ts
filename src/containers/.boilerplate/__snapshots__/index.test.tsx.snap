// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page component should render successfully 1`] = `
<div>
  Boilerplate
</div>
`;
